packages/braid-design-system/src/lib/components/private/InlineField/StyledInput.tsx

```diff
@@ -210,6 +213,10 @@ export const StyledInput = forwardRef<
             variant={disabled ? 'disabled' : defaultBorder}
             borderRadius={fieldBorderRadius}
             visible={tone !== 'critical' || disabled}
+            className={{
+              [styles.hideBorderOnDarkBackgroundInLightMode]:
+                lightMode === 'dark',
```

Comment:
What is `lightMode === 'dark'` conveying here? Just reading it is a bit confusing.

Reply:
`useBackgroundLightness()` returns and object containing whether the current background colour is `light` or `dark`, and for each colour mode.
So the logic is, if the current background colour applied in `lightMode` is a *dark* colour...

Does that make sense?

Reply:
Ah, that's a lot clearer. 👍